Xamarin.Forms.Platform.UAP/Platform.cs

```diff
@@ -46,7 +48,7 @@ public static IVisualElementRenderer CreateRenderer(VisualElement element)
 				throw new ArgumentNullException(nameof(element));
 
 			IVisualElementRenderer renderer = Registrar.Registered.GetHandlerForObject<IVisualElementRenderer>(element) ??
-			                                  new DefaultRenderer();
+											  new DefaultRenderer();
```

Comment:
Not sure about these style changes

Reply:
Ctrl + K 
Ctrl + D

I will review this changes

Reply:
There was a spaces, I replaced them wit tab